### PR TITLE
[SPARK-6083] [MLLib] [DOC] Make Python API example consistent in NaiveBayes

### PR DESCRIPTION
**Pull request by MechCoder**

### CODE REVIEW

**Comment by MechCoder** [comment]
cc: @mengxr Would you be able to verify this?

**Comment by MechCoder** [comment]
Hmm. I get an a accuracy of zero for the given example. Not sure where I'm going wrong though :(

**Comment by MechCoder** [comment]
I changed the randomSplit seed and it works better. It should look good now.

**Comment by mengxr** [comment]
@MechCoder Thanks for the update! We only have 6 lines in `sample_naive_bayes_data.txt`. That's why some random seed would give bad splits.

**Comment by MechCoder** [comment]
Great. Do you have any more comments?

File: docs/mllib-naive-bayes.md [hunk 1]
```diff
@@ -115,22 +115,31 @@ used for evaluation and prediction.
...
-from pyspark.mllib.regression import LabeledPoint
 from pyspark.mllib.classification import NaiveBayes
+from pyspark.mllib.linalg import Vectors
+from pyspark.mllib.regression import LabeledPoint
+
+data = sc.textFile("data/mllib/sample_naive_bayes_data.txt")
+
+# Preprocessing
+splitData = data.map(lambda line: line.split(','))
+parsedData = splitData.map(
```
Comment (mengxr) [inline docs/mllib-naive-bayes.md]: We can define a parse function to make the code more readable. Btw, we use 4 space indentation in Python, following PEP8.

~~~python
def parseLine(line):
    parts = line.split(',')
    label = float(parts[0])
    features = Vector.dense([float(x) for x in parts[1].split(' ')])
    return LabeledPoint(label, features)

data = sc.textFile('data/mllib/sample_naive_bayes_data.txt').map(parseLine)
~~~

File: docs/mllib-naive-bayes.md [hunk 2]
```diff
@@ -115,22 +115,31 @@ used for evaluation and prediction.
...
+    Vectors.dense(map(lambda x: float(x), parts[1].split(' ')))
+    )
+  )
 
-# an RDD of LabeledPoint
-data = sc.parallelize([
-  LabeledPoint(0.0, [0.0, 0.0])
-  ... # more labeled points
-])
+# Split data into training (60%) and test (40%)
```
Comment (mengxr) [inline docs/mllib-naive-bayes.md]: `data into` -> `data approximately into`.

**Comment by MechCoder** [comment]
@mengxr fixed !

**Comment by mengxr** [comment]
LGTM.

**Comment by mengxr** [comment]
Merged into master and branch-1.3. Thanks!

**Pull request closed without merging**